docs/index.rst

```diff
@@ -1,30 +1,142 @@
-====================================
-Welcome to OpenHatch's documentation
-====================================
+====================
+Welcome to OpenHatch
+====================
 
-You've reached the documentation for `OpenHatch <http://openhatch.org/>`_.  
+`OpenHatch <http://openhatch.org/>`_ values open communication. We believe
+documentation is an important part of a welcoming community.
 
-Trying to get your bearings?  Take a look at the :doc:`getting_started/project_overview` or :doc:`community/contact` to ask questions.
-
-We've also got a `wiki <https://openhatch.org/wiki/Main_Page>`_ with a lot of informal documentation and other useful things on it.
+Trying to get your bearings with our documentation?  Take a look at the
+:doc:`getting_started/project_overview` or :doc:`community/contact` to ask
+questions. We also have a `wiki <https://openhatch.org/wiki/Main_Page>`_
+with a lot of informal documentation and other useful things on it.
 
 Confused or dismayed by our documentation?  Please let us know so we can
-improve it!  Make an issue on our `issue tracker <https://github.com/openhatch/oh-mainline/issues>`_ or :doc:`community/contact` via email or IRC to chat with us about it.
+improve it!  Make an issue on our
+`issue tracker <https://github.com/openhatch/oh-mainline/issues>`_
+or :doc:`community/contact` via email or IRC to chat with us about it. We
+take our documentation seriously and wish it to be helpful to you.
 
 
-Content
-=======
+Contents
+========
 
 .. toctree::
-    :maxdepth: 2
+    :maxdepth: 1
 
     getting_started/index
     advanced/index
+    internals/index
     tutorials/index
     community/index
-    internals/index
-    CONTRIBUTING.rst
-    README.rst
 
-:ref:`search`
 
+Contributors Guide
```

Comment:
Probably this should be one of:

* Contributors' Guide
* Contributor's Guide
* Contribution Guide

Python.org goes with the middle one: https://docs.python.org/devguide/ "Developer's Guide"